tests/suites/test_suite_psa_crypto.data

```diff
@@ -1146,10 +1130,6 @@ PSA MAC setup: bad algorithm (HMAC without specified hash)
 # Either INVALID_ARGUMENT or NOT_SUPPORTED would be reasonable here
 mac_setup:PSA_KEY_TYPE_HMAC:"000102030405060708090a0b0c0d0e0f101112131415161718191a1b1c1d1e1f202122232425262728292a2b2c2d2e2f303132333435363738393a3b3c3d3e3f":PSA_ALG_HMAC(0):PSA_ERROR_NOT_SUPPORTED
 
-PSA MAC setup: bad algorithm (unsupported HMAC hash algorithm)
```

Comment:
We still need to have a test with an unsupported HMAC hash algorithm. Please keep this test case with MD5 instead of MD2.